lib/common.js

```diff
@@ -45,8 +45,7 @@ const getSimpleDataType = (type, isCollection) => {
  * @param {String} version the version of the metadata to fetch, for example "1.7" (current default)
  * @returns the latest copy of the metadata file at the URL in the description
  */
-const getMetadata = (version = CURRENT_DD_VERSION) =>
-  require(`./references/dd-${version}/metadata-report.json`) || {};
+const getMetadata = (version = CURRENT_DD_VERSION) => require(`./references/dd-${version}/metadata-report.json`) || {};
```

Comment:
This formatting is confusing compared to the old formatting. Should have some indentation for future reference.